extensions/2.0/Khronos/KHR_avatar_expression_texture/README.md

```diff
@@ -0,0 +1,262 @@
+# KHR_avatar_expression_texture
```

Comment:
My honest impression is that including this extension in Phase 1 feels a bit out of place. Using texture switching to express facial expressions seems relatively rare to me, and I associate it more with toon-style rendering. While glTF can achieve toon rendering (with extensions), I feel its primary target is more on the realistic side. Because of that, this texture-based facial expression approach may not be mainstream, and it might be fine to consider it later.

Is there a specific reason this proposal is in Phase 1? Is it because you see it as a foundation for future extensions?

Reply:
This was a direct request from our partners in the VRM Consortium, and frankly I agree with the sentiment. If we don't include texture-based animations (which is present in a lot of eastern animation), we're effectively marginalizing it. 

It's also important for us to convey that these extensions are not purely for realistic models; but for all types of characters and avatars.  

Totally understand the sentiment FWIW.